frontend/src/components/WofButton.vue

```diff
@@ -87,6 +87,21 @@ export default {
     }
   }
 
+  &--outline {
+    color: black;
+    background-color: transparent;
+    border: 1px solid @primary-dark-text-color;
+    &:hover {
+      background-color: fade(@primary-neutral-color, 30);
```

Comment:
Can you explain what it is supposed to do? I cannot find a fade value for background color (or at all) on the webs.

Reply:
It is simple way to make something partly transparent

Reply:
oh, it's not standard css, it's less: https://lesscss.org/functions/#color-operations-fade understood.